packages/aws-cdk-lib/aws-codebuild/test/codebuild.test.ts

```diff
@@ -1751,12 +1751,12 @@ describe('Linux x86-64 Image', () => {
 });
 
 describe('ARM image', () => {
-  describe('AMAZON_LINUX_2_ARM', () => {
```

Comment:
Since ARM_LINUX2_ARM is deprecated, I took the opportunity to update it as well.